transaction/extra/tests/verifier.rs

```diff
@@ -773,31 +767,35 @@ fn test_sci_three_way_tx_summary_verification() {
         &tx_summary,
         &tx_summary_unblinding_data,
         *bob.view_private_key(),
+        bob.change_subaddress(),
     )
     .unwrap();
     assert_eq!(
         &mlsag_signing_digest[..],
         &signing_data.mlsag_signing_digest[..]
     );
 
-    let balance_changes: Vec<_> = report
-        .balance_changes
-        .iter()
-        .map(|(x, y)| (x.clone(), *y))
-        .collect();
-
     let charlie_hash = ShortAddressHash::from(&charlie.default_subaddress());
-    let mut expected = vec![
-        ((TransactionEntity::Ourself, token2), -(value2 as i64)),
+
+    assert_eq!(
+        &report.totals,
+        &[
+            // Bob's spend to create the transaction
+            (token2, value2 as i64),
+        ]
+    );
+    let mut outputs = vec![
+        // Converted output to charlie, - fee paid from Mob input
         (
-            (TransactionEntity::Address(charlie_hash), Mob::ID),
-            ((value - Mob::MINIMUM_FEE) as i64),
+            TransactionEntity::OtherAddress(charlie_hash),
+            Mob::ID,
+            (value - Mob::MINIMUM_FEE),
         ),
-        ((TransactionEntity::Swap, Mob::ID), -(value as i64)),
-        ((TransactionEntity::Swap, token2), (value2 as i64)),
+        // Output to swap counterparty
```

Comment:
i think this is an example of a report entry created by the line mentioned below as "Note: This isn't exercised"